jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/annotations/AnnotationDiagnosticsCollector.java

```diff
@@ -117,6 +116,7 @@ public void collectDiagnostics(ICompilationUnit unit, List<Diagnostic> diagnosti
                                                     annotationRange,
                                                     "The attribute 'date' of the annotation @Generated must follow the ISO 8601 standard.");
                                             diagnostic.setCode(AnnotationConstants.DIAGNOSTIC_CODE_DATE_FORMAT);
+                                            completeDiagnostic(diagnostic);
```

Comment:
This looks to have code already merged to main - ensure you are working off the latest changes from main (can be accomplished with a `git rebase`)